tests/unit/test_local_session.py

```diff
@@ -172,6 +172,33 @@ def test_describe_model(LocalSession):
     assert response['PrimaryContainer']['ModelDataUrl'] == '/some/model/path'
 
 
+@patch('sagemaker.local.local_session._LocalTransformJob')
+@patch('sagemaker.local.local_session.LocalSession')
+def test_create_transform_job(LocalSession, _LocalTransformJob):
+    local_sagemaker_client = sagemaker.local.local_session.LocalSagemakerClient()
+
+    local_sagemaker_client.create_transform_job('transform-job', 'some-model', None, None, None)
+    _LocalTransformJob().start.assert_called()
+
+    local_sagemaker_client.describe_transform_job('transform-job')
+    _LocalTransformJob().describe.assert_called()
```

Comment:
I'd change both of these to use `assert_called_with()`

Reply:
the 2nd one is a no-arg call (describe()) on a TransformJob Object.